src/types/JobListing.ts

```diff
@@ -0,0 +1,32 @@
+export type Job = {
+  id: number;
+  company: string;
+  logo: string;
+  new: Boolean;
```

Comment:
`new` is a Javascript keyword so I would avoid using it, maybe something like `isNew`?

Reply:
good call...and the same for featured (below comment)